package.json

```diff
@@ -73,5 +73,8 @@
   },
   "lint-staged": {
     "*.{js,ts,tsx}": "eslint --max-warnings 0"
+  },
+  "dependencies": {
+    "react-select": "^4.3.1"
```

Comment:
I think this should be in the /public/package.json

Reply:
I've corrected this in the new commit